src/ComponentBase.ts

```diff
@@ -433,6 +427,18 @@ abstract class ComponentBase<P extends React.Props<any>, S extends Object> exten
         return undefined;
     }
 
+    // Access to the initial state of the component which is unavailable in componentWillMount otherwise
```

Comment:
There is slight ambiguity for how to make use of this method. The component should not call it directly, but rather should override it. It should only be called once and the base class already does that. Perhaps the comment could begin with "Override to gain access" or something.

Reply:
fixed